packages/config-service/src/services/validationService.ts

```diff
@@ -0,0 +1,95 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { GlobalConfig } from './globalConfig';
+
+export class ValidationService {
+  static startUp(envs: NodeJS.Dict<string>): void {
+    // make sure that mandatory fields are passed
+    Object.entries(GlobalConfig.ENTRIES).forEach(([entryName, entryInfo]) => {
+      if (entryInfo.required) {
+        if (!envs[entryName]) {
+          throw new Error(`${entryName} is a mandatory and the relay can not operate without its value.`);
+        }
+
+        if (entryInfo.type === 'number' && isNaN(Number(envs[entryName]))) {
+          throw new Error(`${entryName} must be a valid number.`);
+        }
+      }
+    });
+
+    // make sure that if OPERATOR_KEY_FORMAT is not specified, the provided OPERATOR_KEY_MAIN is in the DER format
+    if (
+      envs[GlobalConfig.ENTRIES.OPERATOR_KEY_MAIN.envName] &&
+      !envs[GlobalConfig.ENTRIES.OPERATOR_KEY_FORMAT.envName] &&
+      // @ts-ignore
+      !envs[GlobalConfig.ENTRIES.OPERATOR_KEY_MAIN.envName].match(/^[0-9a-f]{96,100}$/)
+    ) {
+      throw new Error(
+        `When ${GlobalConfig.ENTRIES.OPERATOR_KEY_FORMAT.envName} is not specified, the ${GlobalConfig.ENTRIES.OPERATOR_KEY_MAIN.envName} must be in DER format.`,
+      );
+    }
+
+    // make sure that HBAR_RATE_LIMIT_TINYBAR is more than HBAR_DAILY_LIMIT_BASIC, HBAR_DAILY_LIMIT_EXTENDED, HBAR_DAILY_LIMIT_PRIVILEGED
```

Comment:
Perhaps `// Ensure HBAR_RATE_LIMIT_TINYBAR is not less than daily limits` to be succinct?